packages/core/services/google-maps-types.ts

```diff
@@ -161,6 +161,7 @@ export interface MapOptions {
   zoom?: number;
   minZoom?: number;
   maxZoom?: number;
+  controlSize: number;
```

Comment:
controlSize is an optional field. Please add 
`controlSize?: number;`

Reply:
Yes, sorry